query-engine/connector-test-kit-rs/query-engine-tests/tests/new/metrics.rs

```diff
@@ -30,6 +30,7 @@ mod metrics {
             Sqlite => assert_eq!(total_queries, 9),
             SqlServer(_) => assert_eq!(total_queries, 15),
             MongoDb(_) => assert_eq!(total_queries, 5),
+            ConnectorVersion::CockroachDb => assert_eq!(total_queries, 10),
```

Comment:
I haven't dug to the bottom of this, but it would make sense for it to be from DROP DATABASE + CREATE DATABASE on test initialization becoming 1 query (instead of 2).